import-scripts/standardize_gene_matrix_file.py

```diff
@@ -0,0 +1,33 @@
+import sys
```

Comment:
Could be useful to add a small comment to the beginning of the script explaining what the script does. I added this to the standardize_clinical_data script from a few weeks ago: https://github.com/knowledgesystems/cmo-pipelines/blob/dc9547064a24d06a0cafb5cf3fe838269f2fd860/import-scripts/standardize_clinical_data.py#L1-L12